test/stress/pa/pa_microbench.c

```diff
@@ -356,11 +382,16 @@ print_shard_stats(int shard_id, size_t operation_count) {
 	}
 
 	/* Dirty bytes */
-	size_t dirty_bytes = psset_stats->merged.ndirty * PAGE;
+	size_t   dirty_bytes = psset_stats->merged.ndirty * PAGE;
+	uint64_t npurge_passes = hpa_stats.nonderived_stats.npurge_passes;
+	uint64_t npurges = hpa_stats.nonderived_stats.npurges;
 
+	assert(psset_stats->merged.nactive * PAGE
+	    == g_shard_stats[shard_id].bytes_allocated);
```

Comment:
does the assertion still work fine if we are using SEC-level simulation? Due to the caching in SEC, we need to use SEC stats to assert here if SEC is involved?